### PR TITLE
Update docs/scripts to use the alpha.11 release packages

### PR DESCRIPTION
**Pull request by daxian-dbw**
Update the docs and scripts to use the alpha.11 release packages.
/cc @andschwa @mirichmo @vors

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Logging to docker/logs/nightly/ubuntu16.04.log
 Waiting for nightly containers to finish; tail the logs for more information.
 
 $ ls packages
-powershell_6.0.0-alpha.10-310-g5ded651-1ubuntu1.16.04.1_amd64.deb
+powershell_6.0.0-alpha.11-310-g5ded651-1ubuntu1.16.04.1_amd64.deb
```
Comment (mirichmo) [inline docker/README.md]: Is the "g5ded651" portion of the filename consistent between releases?

Comment (andschwa) [inline docker/README.md]: Ah this text is just from a quoted example. It doesn't need to be changed (but being an example; there's really no harm in it changing either).

The original text `alpha.10-310-g5ded651` came from having done a `git describe` on a commit that was not tagged (specifically commit g5ded651 which was 310 commits past alpha.10), which is what you're seeing here.

**Review by andyleejordan — APPROVED** [review]
I double-checked, looks like you got them all.

**Comment by daxian-dbw** [comment]
Thanks @andschwa, I reverted the changes to `docker/README.md`.

**Pull request merged**